binaries/bzutil/main.go

```diff
@@ -64,6 +72,20 @@ func main() {
 	uploadJson := uploadCommand.Bool("json", false, "Print command digest as JSON to stdout")
 	uploadLogLevel := uploadCommand.String("log_level", "", "Log everything at this level and above (error|info|debug)")
 
+	// Batch Upload
+	batchUpload := flag.NewFlagSet(batchUploadStr, flag.ExitOnError)
+	batchUploadAddr := batchUpload.String("cas_addr", scootapi.DefaultApiBundlestore_GRPC, "'host:port' of grpc CAS server")
+	batchUploadDir := batchUpload.String("dir", "", "dir containing files to upload: '/dir'")
+	batchUploadTest := batchUpload.String("test", "false", "when true, will upload 10 files randomly ranging from 1k to 1m")
+	batchUploadLogLevel := batchUpload.String("log_level", "info", "Log everything at this level and above (error|info|debug), default info")
+
+	// Batch Download
+	batchDownload := flag.NewFlagSet(batchDownloadStr, flag.ExitOnError)
+	batchDownloadAddr := batchDownload.String("cas_addr", scootapi.DefaultApiBundlestore_GRPC, "'host:port' of grpc CAS server")
+	batchDownloadDigests := batchDownload.String("digests", "", "digests identifying the contents to download: '[{Hash:<hash>, SizeBytes:<size>},...]")
```

Comment:
Last consistency nit, all other digests get supplied in the hash/size format, if json is easier to integrate with the test tool prefer we have 2 flags that specify the digests in either plaintext or json forms